Source/WebKit/UIProcess/API/Cocoa/WKWebExtension.mm

```diff
@@ -510,4 +510,16 @@ - (BOOL)_hasModularBackgroundContent
 
 #endif // ENABLE(WK_WEB_EXTENSIONS)
 
+#if ENABLE(WK_WEB_EXTENSIONS_SIDEBAR)
```

Comment:
would it be better to move this `ENABLE(WK_WEB_EXTENSIONS_SIDEBAR)` case under `ENABLE(WK_WEB_EXTENSIONS)` and the `!ENABLE(WK_WEB_EXTENSIONS_SIDEBAR)` under `!ENABLE(WK_WEB_EXTENSIONS)`?

Reply:
I generally keep them separate because it looks a tiny bit cleaner, and because `ENABLE_WK_WEB_EXTENSIONS_SIDEBAR` depends on `ENABLE_WK_WEB_EXTENSIONS`:

from `PlatformEnableCocoa.h`: `#define ENABLE_WK_WEB_EXTENSIONS_SIDEBAR 0 && ENABLE_WK_WEB_EXTENSIONS`

Reply:
I imagine we would eventually get rid of `ENABLE_WK_WEB_EXTENSIONS_SIDEBAR` when it ships (or at least this is what we do in Safari, not sure if that's the approach we take in WebKit), which is why I think it'd be better to move it within the `ENABLE(WK_WEB_EXTENSIONS)` block

Reply:
Alrighty, just moved this around (and another similar occurrence as well)